src/main/java/webdriver/screen/yandex/disk/LoginPage.java

```diff
@@ -11,6 +14,7 @@
     private static final By FIELD_FOR_LOGIN_LOCATOR = By.id("passp-field-login");
     private static final By BUTTON_LOGIN_LOCATOR = By.xpath("//*[@* = 'submit']");
     private static final By FIELD_FOR_PASSWORD_LOCATOR = By.id("passp-field-passwd");
+    private final Logger logger = LogManager.getRootLogger();
```

Comment:
I'd move the logger to a separate class and just use its static methods to log actions on different levels